src/sympc/module/nn/__init__.py

```diff
@@ -1,8 +1,9 @@
 """Modules and specifc functionalities needed in a Neural Network."""
 
 from .conv import Conv2d
+from .functional import max_pool2d
 from .functional import mse_loss
 from .functional import relu
 from .linear import Linear
 
-__all__ = ["relu", "mse_loss", "Linear", "Conv2d"]
+__all__ = ["relu", "mse_loss", "max_pool2d", "Linear", "Conv2d"]
```

Comment:
Why do we not have an `nn.MaxPool2d` that calls `F.max_pool2d`? It would feel more intuitive to translate PyTorch models to SyMPC models.

[PyTorch src](https://pytorch.org/docs/stable/_modules/torch/nn/modules/pooling.html#:~:text=class%20MaxPool2d(_MaxPoolNd)%3A)

Reply:
We followed the approach with ```nn.functional.max_pool2d``` (pytorch let's you use those functions from more places).